src/mm-formal.adoc

```diff
@@ -882,29 +872,29 @@ generate the new system state.
 ===== Fetch instruction
 
 A possible program-order-successor of instruction instance
-latexmath:[$i$] can be fetched from address _loc_ if:
+_i_ can be fetched from address _loc_ if:
 
 . it has not already been fetched, i.e., none of the immediate
-successors of latexmath:[$i$] in the hart’s _instruction_tree_ are from
+successors of _i_ in the hart’s _instruction_tree_ are from
 _loc_; and
-. if latexmath:[$i$]’s pseudocode has already written an address to
+. if _i_’s pseudocode has already written an address to
 _pc_, then _loc_ must be that address, otherwise _loc_ is:
 * for a conditional branch, the successor address or the branch target
 address;
 * for a (direct) jump and link instruction (`jal`), the target address;
 * for an indirect jump instruction (`jalr`), any address; and
-* for any other instruction, latexmath:[$i.\textit{program\_loc}+4$].
+* for any other instruction, _i.program_loc_+4.
```

Comment:
The underscore inside the italics looks fishy.  You might need to double them up on the outside, e.g. `__i.program_loc__+4`